src/router/kromo/dd-wrt/lang_pack/english.js

```diff
@@ -1328,7 +1328,7 @@ log_out.th_port="Service/Port Number";
 
 // ** Log.asp **//
 var log=new Object();
-log.titl="Log";
+log.label="Firewall Log";
```

Comment:
@sparksofinsanity

`log.titl` was never used anywhere currently, and we were missing a label string anyway. this will be used in Firewall.asp page, this PR will update all languages string.name but not its translation so you are aware for your future work.

Im waiting for your patch to be merged first, because its easier for me to rebase ontop of yours.

I already ponged bs about your patch

thanks again mr insanity sparks